lua/bufferline.lua

```diff
@@ -191,12 +196,12 @@ function bufferline.setup(user_config)
     {count = true, desc = 'Scroll the bufferline right'}
   )
 
-  -- Set the options and watchers for when they are edited
-  vim.g.bufferline = user_config or vim.empty_dict()
+  -- Setup barbar
+  events.enable()
+  vim.g.bufferline = user_config
 
-  highlight.setup()
-  JumpMode.set_letters(options.letters())
-  render.enable()
+  -- Show the tabline
+  set_option('showtabline', 2)
```

Comment:
I moved this to the bottom, so that in case there are any errors during setup the `showtabline` option won't be set (unless it was already set in the user's configuration)